0.6/consul/Dockerfile

```diff
@@ -1,19 +1,16 @@
-FROM alpine:3.2
+FROM alpine:3.4
 
 ENV CONSUL_VERSION 0.6.4
 ENV CONSUL_SHA256 abdf0e1856292468e2c9971420d73b805e93888e006c76324ae39416edcf0627
-ENV GLIBC_VERSION "2.23-r1"
 
-RUN apk --update add curl ca-certificates && \
-    curl -Ls https://github.com/andyshinn/alpine-pkg-glibc/releases/download/${GLIBC_VERSION}/glibc-${GLIBC_VERSION}.apk > /tmp/glibc-${GLIBC_VERSION}.apk && \
-    apk add --allow-untrusted /tmp/glibc-${GLIBC_VERSION}.apk && \
-    rm -rf /tmp/glibc-${GLIBC_VERSION}.apk /var/cache/apk/*
-ADD https://releases.hashicorp.com/consul/${CONSUL_VERSION}/consul_${CONSUL_VERSION}_linux_amd64.zip /tmp/consul.zip
-RUN echo "${CONSUL_SHA256}  /tmp/consul.zip" > /tmp/consul.sha256 \
-  && sha256sum -c /tmp/consul.sha256 \
-  && cd /bin \
-  && unzip /tmp/consul.zip \
-  && chmod +x /bin/consul \
-  && rm /tmp/consul.zip
+RUN apk --no-cache add curl ca-certificates \
+    && curl -sSL https://releases.hashicorp.com/consul/${CONSUL_VERSION}/consul_${CONSUL_VERSION}_linux_amd64.zip -o /tmp/consul.zip \
+    && echo "${CONSUL_SHA256}  /tmp/consul.zip" > /tmp/consul.sha256 \
+    && sha256sum -c /tmp/consul.sha256 \
+    && cd /bin \
+    && unzip /tmp/consul.zip \
+    && chmod +x /bin/consul \
+    && rm /tmp/consul.zip \
+    && apk del curl ca-certificates
```

Comment:
You added "apk del curl ca-certificates", but I think we need to keep curl and ca-certificates for consul healthchecks, right?